src/Groups/GAPGroups.jl

```diff
@@ -626,6 +612,34 @@ end
 # END subgroups conjugation
 
 
+# START iterator
+Base.IteratorSize(::Type{<:GroupConjClass}) = Base.SizeUnknown()
+
+function Base.iterate(cc::GroupConjClass{S,T}) where {S,T}
+  L=GAP.Globals.Iterator(cc.CC)
+  if GAP.Globals.IsDoneIterator(L)
+    return nothing
+  end
+  i = GAP.Globals.NextIterator(L)
+  if T <: GAPGroupElem
+     return group_element(cc.X, i), L
+  else
+     return _as_subgroup(cc.X, i)[1], L
+  end
+end
```

Comment:
Avoid code duplication:
```suggestion
Base.iterate(cc::GroupConjClass) = iterate(CC, GAP.Globals.Iterator(cc.CC))
```